plugins/hls-refactor-plugin/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -1995,11 +1995,19 @@ smallerRangesForBindingExport lies b =
   where
     unqualify = snd . breakOnEnd "."
     b' = wrapOperatorInParens $ unqualify b
+    ranges'
+        ( L
+            _
+            ( IEThingWith
+                _
+                thing
+                _
+                inners
 #if MIN_VERSION_ghc(9,9,0)
-    ranges' (L _ (IEThingWith _ thing _  inners _))
-#else
-    ranges' (L _ (IEThingWith _ thing _  inners))
+                _
 #endif
+            )
+        )
```

Comment:
This arbitrary syntactic rearrangement is for avoiding an error in `stylish-haskell` when it acts on this module:
```
plugins/hls-refactor-plugin/src/Development/IDE/Plugin/CodeAction.hs: <string>:2002:5:
    error: [GHC-58481]
        parse error (possibly incorrect indentation or mismatched brackets)
```